samples/Microsoft.Bot.Samples.Connector.EchoBot/Controllers/MessagesController.cs

```diff
@@ -30,7 +30,7 @@ public MessagesController(IConfiguration configuration)
             var authHeader = this.Request.Headers["Authorization"].SingleOrDefault();
             try
             {
-                await JwtTokenValidation.AssertValidActivity(activity, authHeader, this.credentials);
+                await JwtTokenValidation.EnsureValidActivity(activity, authHeader, this.credentials);
```

Comment:
The useag here is weird. 

The old method was very clearly an Assert(). If it failed, it woudl throw. That's clear. 

This new "Ensure" method, which now returns stuff is being used here as though it's an Assert, and the new Claims Identity isn't being checked.